src/ddscxx/src/org/eclipse/cyclonedds/core/policy/PolicyDelegate.cpp

```diff
@@ -876,6 +876,87 @@ void OwnershipDelegate::set_c_policy(dds_qos_t* qos) const
 }
 
 
+//==============================================================================
```

Comment:
Please also encapsulate this whole body with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro.

Reply:
Will do.